src/components/button-toggle-group/button-toggle-group.stories.mdx

```diff
@@ -21,16 +21,14 @@ Press one of the buttons to make selection. ButtonToggleGroup component should b
 
 ## Quick Start
 
-To use the `ButtonToggleGroup` component you have to import two components: `<ButtonToggleGroup />` and `<ButtonToggle />`:
```

Comment:
Why has this been removed?

Reply:
I've just finished my own review - I noticed this but thought it was fine to remove, it seems a bit of a redundant statement to me, especially as these 2 imports are already shown at the top of the page. But if you think it's better to keep and James agreed, I won't argue :)

Reply:
I'm not _that_ precious about it 😄 but it was more the check it wasn't removed accidentally.

Reply:
This was intentional for the reason @robinzigmond gave 👍🏼